packages/components/src/Layout/Semantics/Semantics.story.tsx

```diff
@@ -25,65 +25,12 @@
  */
 
 import React from 'react'
-import { Story } from '@storybook/react/types-6-0'
 import styled from 'styled-components'
-import { Layout, Header, Aside, AsideProps, Footer, Section } from '..'
-import { Grid } from '../../Grid'
+import { Grid } from '../Grid'
+import { Layout, Header, Aside, Footer, Section } from '.'
 
 export default {
-  title: 'Layout',
-}
-
-const Template: Story<AsideProps> = (args) => (
-  <Aside p="large" text-align="center" {...args} />
-)
-
-export const DefaultBorderColor = Template.bind({})
-DefaultBorderColor.args = {
-  border: true,
-  children: 'Aside',
-}
-
-export const Border = Template.bind({})
-Border.args = {
-  border: 'key',
-  children: 'border',
-}
-
-export const BorderBottom = Template.bind({})
-BorderBottom.args = {
-  borderBottom: 'key',
-  children: 'border-bottom',
-}
-
-export const BorderLeft = Template.bind({})
-BorderLeft.args = {
-  borderLeft: 'key',
-  children: 'border-left',
-}
-
-export const BorderRight = Template.bind({})
-BorderRight.args = {
-  borderRight: 'key',
-  children: 'border-right',
-}
-
-export const BorderTop = Template.bind({})
-BorderTop.args = {
-  borderTop: 'key',
-  children: 'border-top',
-}
-
-export const BorderX = Template.bind({})
-BorderX.args = {
-  borderX: 'key',
-  children: 'border-left and border-right',
-}
-
-export const BorderY = Template.bind({})
-BorderY.args = {
-  borderY: 'key',
-  children: 'border-bottom and border-top',
+  title: 'Semantics',
```

Comment:
To avoid confusion with other Looker use of the word "semantics" (in reference to LookML) let's be more specific with the title here :)

```suggestion
  title: 'Semantic Layout',
```